content/user/admin/site_specific_overrides.md

```diff
@@ -5,6 +5,8 @@ category: admin
 weight: 10
 ---
 
+## Admin Program file 
```

Comment:
Would changing "Program" to "Config" or "Settings" be slightly clearer to the storeowner?

Reply:
I really struggled with this.  Please see 

https://docs.zen-cart.com/user/admin/site_specific_overrides/
https://docs.zen-cart.com/user/customizing/site_specific_overrides/

and let me know your thoughts on better section titles.

Reply:
I realized I had fat fingered the store side titles, so I had to update both files yet again.  I settled on "Settings."   Thanks for the suggestion.